pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -156,7 +156,15 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	// provide any config overrides
 	cfg.NodeName = options.NodeName
-	cfg.StreamingConnectionIdleTimeout = 5 * time.Minute // TODO: should be set
+	streamingConnectionIdletimeout, ok := options.KubeletArguments["streaming-connection-idle-timeout"]
+	if ok && len(options.KubeletArguments["streaming-connection-idle-timeout"]) > 1 {
```

Comment:
set the default we want in the block above where we set lots of other defaults (ending around line 135ish), and then if they set a custom flag value, it'll just override our default